packages/docs/static/themes/medicare-theme.css

```diff
@@ -444,7 +444,6 @@
 --color-warn-darker: #ae8916;
 --color-warn-darkest: #7c6210;
 --color-visited: #590000;
---font-sans: Rubik, Seravek, 'Gill Sans Nova', Ubuntu, Calibri, 'DejaVu Sans', source-sans-pro, sans-serif;
```

Comment:
did a local search for these variables and this one popped up in the `design-system-tokens/src/css/__snapshots__` dir.

Reply:
That snapshot will get updated pretty soon